scripts/settings_list.py

```diff
@@ -0,0 +1,53 @@
+import yaml
+import pprint
+
+class SettingsList():
+  list_of_dicts= list();
+
+  def sl_print(self):
+    pp = pprint.PrettyPrinter(indent=2)
+    pp.pprint(self.list_of_dicts)
+
+  def sl_get_dict(self, group, name):
+    for element in self.list_of_dicts:
+        if element['name'] == name and element['group'] == group:
+          return element
+
+  def sl_get_field(self, group, name,field):
+    thisdict = self.sl_get_dict(group, name)
+    returnvar = ""
+    if thisdict and isinstance(thisdict, dict):
+      returnvar = thisdict.get(field,"")
+    else:
+      print "Error in settings list parsed yaml file."
+      print "No dictionary dictionary for name  {0} and group is {1}".format(name,group)
```

Comment:
Maybe change "dictionary dictionary" for something more user friendly like "entry"?